### PR TITLE
Game play tests

### PR DESCRIPTION
**Pull request by cesine**

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -204,15 +204,20 @@ function App() {
     }
     console.log('input', input, 'ariaLabel', ariaLabel)
     const label = ariaLabel || currentLetter
-    if (label && input && currentCipher[label]) {
+    if (
+      label &&
+      input &&
+      currentCipher[label] &&
+      input !== currentCipher[label].guesses[0]
```
Comment (cesine) [inline src/App.tsx:211]: avoid adding guesses if have the same event replaying

File: src/App.tsx [hunk 2]
```diff
@@ -204,15 +204,20 @@ function App() {
...
+    ) {
       const updatedCipher = { ...currentCipher }
 
       updatedCipher[label].guesses = [input, ...updatedCipher[label].guesses]
       setGuesses([...guesses, input])
       console.log('updated updatedCipher', updatedCipher)
       setCurrentCipher(updatedCipher)
     }
-    if (!input && ariaLabel) {
+    if (ariaLabel) {
```
Comment (cesine) [inline src/App.tsx:220]: this makes it so we can undo when we click elsewhere

File: playwright.config.ts [hunk 3]
```diff
@@ -29,6 +29,9 @@ export default defineConfig({
     /* Base URL to use in actions like `await page.goto('/')`. */
     baseURL: BASE_URL,
 
+    /* Show the browser if we are not in CI */
+    headless: !process.env.CI,
```
Comment (cesine) [inline playwright.config.ts:33]:
```suggestion
    headless: process.env.CI,
```

File: playwright.config.ts [hunk 4]
```diff
@@ -29,6 +29,9 @@ export default defineConfig({
     /* Base URL to use in actions like `await page.goto('/')`. */
     baseURL: BASE_URL,
 
+    /* Show the browser if we are not in CI */
+    headless: process.env.CI,
```
Comment (mecathcart) [inline playwright.config.ts:33]:
```suggestion
    headless: !!process.env.CI,
```

**Pull request merged**